js/simple-pig-latin.test.js

```diff
@@ -0,0 +1,15 @@
+const pigIt = require('./simple-pig-latin');
+
+
+describe('simple-pig-latin', () => {
+
+  test('When it get Pig latin is cool it return  igPay atinlay siay oolcay', () => {
+    expect(pigIt('Pig latin is cool')).toBe('igPay atinlay siay oolcay');
+  });
+
+  test('When it get Pig latin is cool ?  it return  igPay atinlay siay oolcay ? test for special character in string', () => {
```

Comment:
You do not have to be as precise in the test name. It could say

```suggestion
  test('should not append -ay to punctuation', () => {
```